src/components/VirtualizedList/__stories__/VirtualizedList.stories.mdx

```diff
@@ -2,13 +2,47 @@ import VirtualizedList from "../VirtualizedList";
 import Button from "../../Button/Button";
 import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
 import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
-import { generateItems, itemRenderer } from "./virtualizedList.components";
+import { generateItems, itemRenderer } from "./virtualizedList.components"; import {useCallback} from "react"; import {
+    overviewPlaySuite
+} from "components/VirtualizedList/__tests__/virtualized-list-interactions-tests";
 
 <Meta title="Navigation/VirtualizedList" component={VirtualizedList} />
 
 <!--- Component template -->
 
-export const virtualizedListTemplate = createComponentTemplate(VirtualizedList);
+export const virtualizedListTemplate = (args) => {
+    const itemRenderer = useCallback((item, index, style) => {
+        const backgroundColor = index % 2 === 0 ? "white" : "#f8f8f0";
+        return (
+            <div key={index} style={style}>
+                <div
+                    style={{
```

Comment:
consider extracting styles to scss?

Reply:
can't be done. this is a dynamic css which position the visible elements in the virtualized list in the right position according to the scroll.

Reply:
I see it now. if you want, you can move all the other rules to a css class, and keep only the width here (like you probably would have done in a "real" component). but it's not critical :)